_pages/best-practices.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+permalink: best-practices/
+---
+
+<h1>Best practices</h1>
+<h2>Realtime updates</h2>
+
+<p>There is need to display actual information to client on buy, sell or expiration of contract. </p>
+
+<p>It is possible to use API-methods like
+    <a href="/api#portfolio"><code>portfolio</code></a>,
+    <a href="/api#profit_table"><code>profit_table</code></a>,
+    <a href="/api#statement"><code>statement</code></a>,
+    <a href="/api#balance"><code>balance</code></a>,
+but all they are "poll-model", i.e. slow and impose additional load on <a href="https://binary.com">Binary.com</a>,
```

Comment:
Consider rephrasing `but they are "poll-model"` to `but these methods are based on polling`

Reply:
If I send {"balance":1, "subscribe":1} then my app doesn't have to keep polling.

Are you saying that binary.com implements this internally by polling?

Reply:
The calls portfolio, profit_table and statement do not permit subscribing to updates.
The call balance is out of place in this list.